docs/cache/manage/limits.md

```diff
@@ -27,61 +27,63 @@ The limits on this page are soft limits that can be altered unless specifically
-| SortedSetPutElement      | 1                                                               |
-| SortedSetPutElements     | Number of elements in request                                   |
-| SortedSetFetchByRank     | Number of elements in response, or 1 if sorted set is not found |
-| SortedSetFetchByScore    | Number of elements in response, or 1 if sorted set is not found |
-| SortedSetGetScore        | 1                                                               |
-| SortedSetGetScores       | Number of elements in request                                   |
-| SortedSetRemoveElement   | 1                                                               |
-| SortedSetRemoveElements  | Number of elements in request                                   |
-| SortedSetGetRank         | 1                                                               |
-| SortedSetIncrementScore  | 1                                                               |
-| SortedSetLength          | 1                                                               |
-| SortedSetLengthByScore   | 1                                                               |
+| API Name                 | Batch API | Operations                                                        |
+| ------------------------ | ----      | ------------                                                      |
+| Set                      |           | 1                                                                 |
+| Get                      |           | 1                                                                 |
+| Delete                   |           | 1                                                                 |
+| Increment                |           | 1                                                                 |
+| Ping                     |           | 1                                                                 |
+| ItemGetType              |           | 1                                                                 |
+| KeyExists                |           | 1                                                                 |
+| KeysExist                | X         | Number of keys in request/2                                       |
+| SetIfNotExists           |           | 1                                                                 |
+| UpdateTtl                |           | 1                                                                 |
+| IncreaseTtl              |           | 1                                                                 |
+| DecreaseTtl              |           | 1                                                                 |
+| ItemGetTtl               |           | 1                                                                 |
+| DictionaryFetch          | X         | Number of fields in response/2, or 1 if dictionary is not found   |
```

Comment:
in a few places in the docs (language support matrix etc.) we are using emoji checkmarks in tables:

https://github.com/momentohq/public-dev-docs/blob/main/plugins/example-code-snippets/src/api-support/api-support-matrix-generator.ts#L520

up to you if you wanna do that.

Reply:
nice! I like that. Updated in [b92305a](https://github.com/momentohq/public-dev-docs/pull/602/commits/b92305abbe45aed100312dbda5625ffb334393ae)